### PR TITLE
Fix file glob for adding sha256sum upload

### PR DESCRIPTION
**Pull request by babaric-dev**

### CODE REVIEW

**Comment by babaric-dev** [comment]
Btw, possbly we dont need to show the sha256sum in releases if we can download it(?).

File: .github/workflows/bootstrap_archives.yml [hunk 1]
```diff
@@ -71,11 +71,20 @@ jobs:
...
+          file: "*.zip"
+          file_glob: true
+          release_name: "Bootstrap archives for Termux application"
+          tag: ${{ steps.get_tag.outputs.tag_name }}
+          body: ${{ steps.get_checksums.outputs.checksums }}
+      - name: Publish sha256 checksum on GitHub
+        uses: termux/upload-release-action@v3.0.3
+        with:
+          repo_token: ${{ secrets.GITHUB_TOKEN }}
+          file: "*.sha256sum"
```
Comment (agnostic-apollo) [inline .github/workflows/bootstrap_archives.yml:87]: There is no need to enable glob here, only a single file `bootstraps.sha256sum` needs to be uploaded. Moreover, don't think `release_name` and `body` is going to be required for second upload, `tag` should be enough.

**Comment by agnostic-apollo** [comment]
> Btw, possbly we dont need to show the sha256sum in releases if we can download it(?).

Na, keep it. Won't have to download if just copying hashes for `termux-app`, etc.

File: .github/workflows/bootstrap_archives.yml (hunk 1 above)
Comment (babaric-dev) [reply .github/workflows/bootstrap_archives.yml]: Done editing! Please check. I will mark this comment as resolved too.

**Comment by babaric-dev** [comment]
I have done all edits. Please merge.

**Comment by agnostic-apollo** [comment]
Closed in e1b1509c

**Pull request closed without merging**